### PR TITLE
Fix save button is still disabled after changing logo settings

### PR DESCRIPTION
**Pull request by chihsuan**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR fixes a regression in that the save button is still disabled after changing logo settings.

Before:

https://github.com/woocommerce/woocommerce/assets/4344253/4dbf9b7a-cff8-42dc-a32b-a92c49460cb2

After:

https://github.com/woocommerce/woocommerce/assets/4344253/fe531008-1b49-4c30-b483-51eac3ad72d9

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new WooCommerce installation with this version.
2. Make sure to enable `customize-store` feature flag
3. Go to `/wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store%2Fassembler-hub`
4. Go to "Add your logo"
5. Upload a logo
6. Click on Save button
7. Change logo width
8. Observe that the "Save" button is clickable.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/block-editor.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const MAX_PAGE_COUNT = 100;
 export const BlockEditor = ( {} ) => {
 	const history = useHistory();
 	const settings = useSiteEditorSettings();
-	const [ blocks, onChange ] = useEditorBlocks();
+	const [ blocks, , onChange ] = useEditorBlocks();
```
Comment (chihsuan) [inline plugins/woocommerce-admin/client/customize-store/assembler-hub/block-editor.tsx:69]: https://github.com/woocommerce/woocommerce/blob/061f63abd9b16ae629b22c1eaf310c48fad2ffc0/plugins/woocommerce-admin/client/customize-store/assembler-hub/hooks/use-editor-blocks.ts#L40-L43

Comment (adrianduffell) [inline plugins/woocommerce-admin/client/customize-store/assembler-hub/block-editor.tsx:69]: Nitpick: suggestion from chatgpt to make this a bit clearer:
```suggestion
	const [ blocks, /* skippedValue */, onChange ] = useEditorBlocks();
```

**Review by adrianduffell — APPROVED** [review]
It works great @chihsuan, thank you! I just left an optional suggestion 🚀

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/block-editor.tsx (hunk 1 above)
Comment (adrianduffell) [reply plugins/woocommerce-admin/client/customize-store/assembler-hub/block-editor.tsx:69]: On second thought I think this just as confusing 🤣

**Pull request merged**